package com.blog.app.Exceptions;

public class UserFoundException {

}
